### ISSUE TITLE
Upgrade Failure

### ISSUE THREAD

**Issue by vClouds** [mautic/mautic#4867]
What type of report is this:

| Q  |

| Bug report? | 

## Description:
Failed upgrade from v2.9.2 and the same error repeats upon fresh install with the error
`The site is currently offline due to encountering an error. If the problem persists, please contact the system administrator.`

## If a bug:
[mautic_prod-2017-09-03.php.txt](https://github.com/mautic/mautic/files/1282665/mautic_prod-2017-09-03.php.txt)
[prod-2017-09-06.php.txt](https://github.com/mautic/mautic/files/1282666/prod-2017-09-06.php.txt)

```
sudo php app/console cache:clear
 // Clearing the cache for the prod environment with debug                      
 // false                                                                       

PHP Fatal error:  Class 'Twig\Loader\FilesystemLoader' not found in /home/mautic/vendor/symfony/twig-bundle/Loader/FilesystemLoader.php on line 26
                                                                               
  [Symfony\Component\Debug\Exception\ClassNotFoundException]                   
  Attempted to load class "FilesystemLoader" from namespace "Twig\Loader".     
  Did you forget a "use" statement for e.g. "Symfony\Component\Templating\Loa  
  der\FilesystemLoader" or "Symfony\Bundle\FrameworkBundle\Templating\Loader\  
  FilesystemLoader"?
```

| Q   | A
| --- | ---
| Mautic version | 2.10.0
| PHP version | 7.1.9

### Steps to reproduce:
1. 
2. 
 
### Log errors: 
Attached are the logs files while upgrading while no log were generated for fresh install
_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by Vrutin** [comment]
Mautic broke for me too after upgrading to 2.10.0

**Comment by dbhurley** [comment]
Fixed with 2.11
